tests/core/files/overdrive/has_sample.json

```diff
@@ -1 +1,189 @@
+    "fullDescription": "<p>\n<p>Can marriage for money lead to love? Find out with the Brides of Convenience bundle!  Includes <i>The Lawyer's Contract Marriage</i> by Amanda Browning, <i>A Convenient Wife</i> by Sara Wood, <i>The Italian's Virgin Bride</i> by Trish Morey, and <i>The Mediterranean Husband</i> by Catherine Spencer.</p>\n",
+    "starRating": 2.0,
+    "popularity": 87,
+    "subjects": [
+        {
+            "value": "Fiction"
+        },
+        {
+            "value": "Romance"
+        }
+    ],
+    "keywords": [
+        {
+            "value": "Passion"
+        }
+    ],
+    "formats": [
+        {
+            "id": "ebook-pdf-adobe",
+            "name": "Adobe PDF eBook",
+            "fileName": "BridesofConvenienceBundle9781426803697",
+            "identifiers": [
+                {
+                    "type": "ISBN",
+                    "value": "9781426803697"
+                }
+            ],
+            "fileSize": 2764930,
+            "partCount": 0,
+            "onSaleDate": "07/01/2007",
+            "rights": [
+                {
+                    "type": "Copying",
+                    "value": 0
+                },
+                {
+                    "type": "Printing",
+                    "value": 0
+                },
+                {
+                    "type": "Lending",
+                    "value": 0
+                },
+                {
+                    "type": "ReadAloud",
+                    "value": 0
+                },
+                {
+                    "type": "ExpirationRights",
+                    "value": 0
+                }
+            ]
+        },
+        {
+            "id": "ebook-epub-adobe",
+            "name": "Adobe EPUB eBook",
+            "fileName": "BridesofConvenienceBundle9781426803697",
+            "identifiers": [
+                {
+                    "type": "ISBN",
+                    "value": "9781426803697"
+                }
+            ],
+            "fileSize": 820171,
+            "partCount": 0,
+            "onSaleDate": "07/01/2007",
+            "rights": [
+                {
+                    "type": "Copying",
+                    "value": 0
+                },
+                {
+                    "type": "Printing",
+                    "value": 0
+                },
+                {
+                    "type": "Lending",
+                    "value": 0
+                },
+                {
+                    "type": "ReadAloud",
+                    "value": 0
+                },
+                {
+                    "type": "ExpirationRights",
+                    "value": 0
+                }
+            ],
+            "samples": [
+                {
+                    "source": "From the book",
+                    "formatType": "ebook-epub-adobe",
+                    "url": "http://excerpts.contentreserve.com/FormatType-410/1071-1/9BD/24F/82/BridesofConvenienceBundle9781426803697.epub"
```

Comment:
Hmmm... This sample, based on the `formatType` seems like it should be adobe protected... but the link to an epub doesn't look protected.